selenium_tests/cross_listing/cross_listing_base_test_case.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from urlparse import urljoin
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_common.pin.page_objects.pin_login_page_object \
+    import PinLoginPageObject
+
+from selenium_tests.account_admin.page_objects.account_admin_dashboard_page_object \
+    import AccountAdminDashboardPage
+from selenium_tests.cross_listing.page_objects.cross_listing_index_page \
+    import IndexPagePageObject
+
+
+class CrossListingBaseTestCase(BaseSeleniumTestCase):
```

Comment:
Re: "A lot of this class shares similar code with the CourseInfoBaseTestCase, it would be good to (eventually) make it all DRYer."   I can't easily get it to work, so I will tinker around it with but will see if I can get a working version of it in a later round (not this round).